packages/dds/merge-tree/src/client.ts

```diff
@@ -304,7 +304,7 @@ export class Client extends TypedEventEmitter<IClientEvents> {
 		accum: TClientData,
 		splitRange?: boolean,
 	): void;
-	public walkSegments<undefined>(
+	public walkSegments<T>(
```

Comment:
Why did this change? Shouldn't `T` be used somewhere?

Reply:
I think I had a linter error that I was trying to get rid of - I'll revert.